Shared/UI/Views/MessageBubbleView.swift

```diff
@@ -12,29 +12,29 @@ class MessageBubbleView: SpeechBubbleView {
     
     let shapeMask = CAShapeLayer()
     let backgroundLayer = CAShapeLayer()
-    let gradientLayer = GradientLayer(with: [.D4TopLeft, .D4BottomRight], startPoint: .topLeft, endPoint: .bottomRight)
+    let darkGradientLayer = GradientLayer(with: [.D4TopLeft, .D4BottomRight],
+                                      startPoint: .topLeft, endPoint: .bottomRight)
+    let lightGradientLayer = GradientLayer(with: [.D4TopLeft, .L4BottomRight],
+                                           startPoint: .topLeft, endPoint: .bottomRight)
```

Comment:
I have to different gradients that fade in and out based upon how focused the the stack is.